src/components/network/network-map.js

```diff
@@ -346,7 +343,7 @@ const NetworkMap = (props) => {
                 network: props.network,
                 updatedLines: props.updatedLines,
                 geoData: props.geoData,
-                useName: props.useName,
+                useName: useName,
```

Comment:
It seems we don't need useName in LineLayer... maybe we could clean this as well ?